packages/wagmi/src/utils/helpers.ts

```diff
@@ -54,7 +54,17 @@ export function getTransport({ chain, projectId }: { chain: Chain; projectId: st
   }
 
   return fallback([
-    http(`${RPC_URL}/v1/?chainId=${ConstantsUtil.EIP155}:${chain.id}&projectId=${projectId}`),
+    http(`${RPC_URL}/v1/?chainId=${ConstantsUtil.EIP155}:${chain.id}&projectId=${projectId}`, {
+      /*
+       * The Blockchain API uses "Content-Type: text/plain" to avoid OPTIONS preflight requests
+       * It will only work for viem >= 2.17.7
```

Comment:
@zoruka If users use `<2.17.7`, they won't be able to set custom header - in this case will it affect any of the features?

Reply:
what will happen is that the OPTIONS preflight request will be sent, but it doesn't affect the behavior of the feature